Mathlib/NumberTheory/NumberField/CanonicalEmbedding/FundamentalCone.lean

```diff
@@ -330,6 +341,163 @@ def quotIntNorm :
 @[simp]
 theorem quotIntNorm_apply (a : integralPoint K) : quotIntNorm ⟦a⟧ = intNorm a := rfl
 
+variable (K) in
+/-- The map that sends an element of `a : integralPoint K` to the associates class
+of its preimage in `(𝓞 K)⁰`. By quotienting by the kernel of the map, which is equal to the
+subgroup of torsion, we get the equivalence `integralPointQuotEquivAssociates`. -/
+def integralPointToAssociates (a : integralPoint K) : Associates (𝓞 K)⁰ :=
+  ⟦preimageOfIntegralPoint a⟧
+
+@[simp]
+theorem integralPointToAssociates_apply (a : integralPoint K) :
+    integralPointToAssociates K a = ⟦preimageOfIntegralPoint a⟧ := rfl
+
+variable (K) in
+theorem integralPointToAssociates_surjective :
```

Comment:
If this is the only place where you need `K` explicit in `integralPointToAssociates` we can maybe make it implicit and use `(K := K)` here.

Reply:
No, it is needed for `K` to be explicit also in some proofs

Reply:
Ok no problem.